shuup/admin/modules/service_providers/views/_edit.py

```diff
@@ -104,27 +102,11 @@ def get_toolbar(self):
         return toolbar
 
 
-class _FormInfoMap(OrderedDict):
-    def __init__(self, form_classes):
-        form_infos = (_FormInfo(formcls) for formcls in form_classes)
-        super(_FormInfoMap, self).__init__(
-            (form_info.choice_value, form_info) for form_info in form_infos)
-
-    def get_by_object(self, obj):
-        return first(
-            fi for fi in self.values() if isinstance(obj, fi.model))
-
-    def get_by_choice_value(self, choice_value):
-        return self.get(choice_value)
-
-    def get_type_choices(self):
-        return [(x.choice_value, x.choice_text) for x in self.values()]
+# TODO (2.0): Remove
+class _FormInfoMap(FormInfoMap):
+    pass
 
 
-class _FormInfo(object):
-    def __init__(self, form_class):
-        self.form_class = form_class
-        self.model = form_class._meta.model
-        modelmeta = self.model._meta
-        self.choice_value = modelmeta.app_label + '.' + modelmeta.model_name
-        self.choice_text = modelmeta.verbose_name.capitalize()
+# TODO (2.0): Remove
+class _FormInfo(FormInfo):
+    pass
```

Comment:
Don't add these underscored classes here. No need to have them here, since they're private: Nobody shouldn't be using them anyway.